### PR TITLE
Improve grammar and wording

### PR DESCRIPTION
**Pull request by akxcv**

### CODE REVIEW

File: lib/verifly/class_builder.rb [hunk 1]
```diff
@@ -1,27 +1,28 @@
 # frozen_string_literal: true
 
 module Verifly
-  # ClassBuilder is something like Uber::Builder, but it
-  # allows the child classes to decide whether they will be used.
-  # I find it much more OOPish
+  # ClassBuilder is similar to Uber::Builder, but it
+  # allows child classes to decide whether they will be used.
+  # I find it much more object-oriented
```
Comment (JelF) [inline lib/verifly/class_builder.rb:6]: i like OOPish, but up to you

File: lib/verifly/verifier.rb [hunk 2]
```diff
@@ -80,7 +80,7 @@ def verify!(context = {})
     private
 
     # @abstract
-    #   implement it like `super { Message.new(status, text, description) }`
+    #   implementation example: `super { Message.new(status, text, description) }`
```
Comment (JelF) [inline lib/verifly/verifier.rb]: it should not be just an example, it should someway force

File: lib/verifly/verifier/applicator_with_options_builder.rb [hunk 3]
```diff
@@ -7,9 +7,9 @@ class Verifier
     # @attr base [Class]
     #   class for which applicator_with_options should be built
     # @attr args [Array]
-    #   array of arguments Verifier.verify invoked with
+    #   array of arguments to invoke Verifier.verify with
```
Comment (JelF) [inline lib/verifly/verifier/applicator_with_options_builder.rb]: It is already invoked, isnt it?

File: lib/verifly/version.rb [hunk 4]
```diff
@@ -3,10 +3,10 @@
...
-  # Semantical naming: 0.a.b.c where
-  # * 0 is for unreleased
-  # * a is for public api changes
-  # * b is for private api changes
-  # * c is for changes, not changing private api
+  # Semantic versioning: 0.a.b.c where
+  # * 0 stands for unreleased
+  # * a stands for public api changes
+  # * b stands for private api changes
+  # * c stands for patch changes (not touching public or private api)
```
Comment (JelF) [inline lib/verifly/version.rb:10]: I think patch should be reworded to incremental or something like this

**Review by JelF — CHANGES_REQUESTED** [review]
Please, run `$ bin/rake` before each commit or at least each push

File: lib/verifly/class_builder.rb (hunk 1 above)
Comment (akxcv) [reply lib/verifly/class_builder.rb:6]: "object-oriented" feels much better to me

File: lib/verifly/verifier.rb (hunk 2 above)
Comment (akxcv) [reply lib/verifly/verifier.rb]: What exactly should be forced?

File: lib/verifly/version.rb (hunk 4 above)
Comment (akxcv) [reply lib/verifly/version.rb:10]: According to semver.org, it's called "patch" version.
"A change that doesn't touch public/private API" is precisely a "patch".

**Comment by akxcv** [comment]
@JelF everything should be resolved now.

**Review by JelF — APPROVED** [review]

**Pull request merged**